services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -430,7 +430,7 @@ func (r *repository) ProcessQueueOnce(ctx context.Context, t Transformer, callba
 	return nil
 }
 
-func (r *repository) ApplyTransformersInternal(ctx context.Context, transaction *sql.Tx, transformers ...Transformer) ([]string, *State, []*TransformerResult, *TransformerBatchApplyError) {
+func (r *repository) ApplyTransformersInternal(ctx context.Context, transaction *sql.Tx, transformer Transformer) ([]string, *State, []*TransformerResult, *TransformerBatchApplyError) {
```

Comment:
Wasn't this already done in #1763?

Reply:
yes, I accidentally pushed it to 2 branches. Fixed.